modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -9,6 +9,16 @@
 
 _XDCR filtering_ allows a document to be included in or excluded from a filtered replication, based on the document's fields and values.
 
+XDCR applies the filter during replication processing, as the data is passed through the replication pipeline.
+Filter-evaluation is performed only in the following circumstances:
+
+* A document is either created or modified.
+
+* A replication is restarted for all documents.
+
+Note that filter-evaluation is _not_ performed based on the measurement of elapsed time: therefore, the replication of a document cannot be triggered by establishing a filter on a time-value within the document.
```

Comment:
Since the time value doesn't have to be numeric -- can be any date, date time value, it might be better to use the term "date time value" instead of "time-value":
instead of "filter on a time-value within the document",
"filter on a date time value within the document"

Reply:
Ok, I've changed.

Reply:
Thanks!